index.js

```diff
@@ -74,10 +74,13 @@ module.exports = ExportNodeModules;
  * @return {String}
  */
 function getLicenses(packageJson) {
-  if (packageJson.licenses) {
+  if (packageJson.licenses && packageJson.licenses instanceof Array) {
     return packageJson.licenses
       .map(license => license.type)
       .join(', ');
+  } else if (packageJson.licenses) {
+    return (packageJson.licenses && packageJson.licenses.type) ||
```

Comment:
Note that here I'm returning `packageJson.licenses` (plural) as opposed to `packageJson.license` (singular). Initially I was going to just reassign `packageJson.license` directly, but that mutates the original. Thoughts?